src/Raven.Server/Documents/DocumentsStorage.cs

```diff
@@ -176,27 +203,48 @@ full name
                 IsGlobal = true,
                 Name = TombstonesSlice
             });
-
             TombstonesSchema.DefineFixedSizeIndex(new TableSchema.FixedSizeSchemaIndexDef
             {
                 StartIndex = 1,
                 IsGlobal = false,
                 Name = CollectionEtagsSlice
             });
-
             TombstonesSchema.DefineFixedSizeIndex(new TableSchema.FixedSizeSchemaIndexDef
             {
                 StartIndex = 1,
                 IsGlobal = true,
                 Name = AllTombstonesEtagsSlice
             });
-
             TombstonesSchema.DefineFixedSizeIndex(new TableSchema.FixedSizeSchemaIndexDef()
             {
                 StartIndex = 2,
                 IsGlobal = false,
                 Name = DeletedEtagsSlice
             });
+
+
+            // The attachments schema is as follows
+            // 5 fields (lowered document id AND record separator AND lowered name, etag, name, content type, last modified, stream identifier)
+            // We are you using the record separator in order to avoid loading another files that has the same key prefix, 
+            //      e.g. fitz(record-separator)profile.png and fitz0(record-separator)profile.png, without the record separator we would have to load also fitz0 and filter it.
+            // format of lazy string key is detailed in GetLowerKeySliceAndStorageKey
+            AttachmentsSchema.DefineKey(new TableSchema.SchemaIndexDef
+            {
+                StartIndex = (int)AttachmentsTable.LoweredDocumentIdAndRecordSeparatorAndLoweredName,
+                Count = 1,
+                Name = AttachmentsKeySlice
+            });
+            AttachmentsSchema.DefineIndex(new TableSchema.SchemaIndexDef
```

Comment:
We should support table.SeekForwardFrom(AttachmentsSchema.Key, startSlice, true) but it is failing now because of expecting fixedSizeTree